### PR TITLE
missing release dates

### PR DESCRIPTION
**Pull request by elopezanaya**
# PR Details

### Description

_Include a description of the problem to be solved_

## Solution Proposed

_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

### Acceptance criteria

_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence

_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### Sanity Tests

- [ ] You have tested all changes in Popout mode
- [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
- [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y

- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

***Please provide justification if any of the validations has been skipped.***

### CODE REVIEW

**Review by Nathanael-ms — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -106,6 +106,35 @@ New releases are published on a regular basis to ensure the product quality.
 
 | Version | Docs | Release Date | End of Support | Deprecated |
 | -- | -- | -- | -- | -- |
+| 1.11.4 | [Release Notes](https://github.com/microsoft/omnichannel-chat-sdk/releases/tag/v1.11.4) | Jul 17th 2025 | Jul 17th 2026 | |
```
[the remark this answers is not in the document]
Comment (dleblondMSFT) [reply README.md:109]: At least, going forward with 1.9.x and beyond

**Review by mithunjohny — APPROVED** [review]
